README.md

```diff
@@ -129,15 +129,24 @@ those? Of course!
 
 The `scenic:view` and `scenic:model` generators accept a `--materialized`
 option for this purpose. When used with the model generator, your model will
-have the following method defined as a convenience to aid in scheduling
+have the following methods defined as conveniences to aid in scheduling
 refreshes:
 
 ```ruby
 def self.refresh
   Scenic.database.refresh_materialized_view(table_name)
 end
+
+def self.refresh_concurrently
```

Comment:
What do you think about making this part of the existing `refresh` method? Something like:

```ruby
def self.refresh(concurently: false)
  Scenic.database.refresh_meterialzed_view(**args)
end
```